pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -86,6 +87,16 @@ def test_multiindex(self):
         assert result == expected
 
 
+class TestParseSchema(object):
```

Comment:
Shouldn't need a new class here - can you just move this to `TestTableOrientReader`?